controllers/flp/flp_pipeline_builder.go

```diff
@@ -89,6 +89,22 @@ func (b *PipelineBuilder) AddProcessorStages() error {
 				AddZone:         addZone,
 			},
 		},
+		{
+			Type: api.NetworkAddKubernetes,
+			Kubernetes: &api.K8sRule{
+				IPField: "XlatSrcAddr",
+				Output:  "XlatSrcK8S",
+				AddZone: addZone,
+			},
+		},
+		{
+			Type: api.NetworkAddKubernetes,
+			Kubernetes: &api.K8sRule{
+				IPField: "XlatDstAddr",
+				Output:  "XlatDstK8S",
+				AddZone: addZone,
+			},
+		},
```

Comment:
Zone (which is about cloud AZ) is defined out of the main IPs not the xlated ones.

```suggestion
		{
			Type: api.NetworkAddKubernetes,
			Kubernetes: &api.K8sRule{
				IPField: "XlatSrcAddr",
				Output:  "XlatSrcK8S",
			},
		},
		{
			Type: api.NetworkAddKubernetes,
			Kubernetes: &api.K8sRule{
				IPField: "XlatDstAddr",
				Output:  "XlatDstK8S",
			},
		},
```